packages/panzer/disc-fe/src/Panzer_FaceToElement.hpp

```diff
@@ -82,27 +82,21 @@ class FaceToElement {
     */
   void initialize(panzer::ConnManager<LocalOrdinal,GlobalOrdinal> & conn);
 
-  KOKKOS_INLINE_FUNCTION
```

Comment:
Is the `KOKKOS_INLINE_FUNCTION` macro no bueno anymore?

Reply:
You still need the macro for true kokkos functions but these functions are impossible to call on device (and for our use cases were only called from host code). They try to use RCPs on device and then call host-only functions. This generated warnings aout calling host code from host/device functions.

Reply:
Gotcha.   I saw the PR before reading through the corresponding email chain.  Makes sense.